tests/profiles/test_profile.py

```diff
@@ -84,6 +84,9 @@ async def test_is_profile_device(self) -> None:
         no_services = await factory.async_create_device(
             "http://dlna_dmr:1234/device_incomplete.xml"
         )
+        empty_descriptor = await factory.async_create_device(
```

Comment:
This is failing because the `factory` is in strict mode, so rejects the bad device.

It's a good idea to test `is_profile_device` for devices that get through a non-strict factory. Perhaps duplicate this function (say `test_is_profile_device_non_strict`) and change it to create the `factory = UpnpFactory(requester, non_strict=True)`.